color_force_grayscale gives error with cadence > 1 and NOT hybrid mode

It works fine with hybrid mode, but gives this error in 2d/3d mode (only with cadence larger than 1):
```
deforum_helpers\render.py", line 260, in render_animation
    img = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
cv2.error: OpenCV(4.7.0) d:\a\opencv-python\opencv-python\opencv\modules\imgproc\src\color.simd_helpers.hpp:94: error: (-2:Unspecified error) in function '__cdecl cv::impl::`anonymous-namespace'::CvtHelper<struct cv::impl::`anonymous namespace'::Set<3,4,-1>,struct cv::impl::A0x39480f9d::Set<1,-1,-1>,struct cv::impl::A0x39480f9d::Set<0,2,5>,2>::CvtHelper(const class cv::_InputArray &,const class cv::_OutputArray &,int)'
> Unsupported depth of input image:
>     'VDepth::contains(depth)'
> where
>     'depth' is 6 (CV_64F)
```

--- a/scripts/deforum_helpers/render.py
+++ b/scripts/deforum_helpers/render.py
@@ -257,7 +257,7 @@ def render_animation(args, anim_args, video_args, parseq_args, loop_args, animat
 
                 # intercept and override to grayscale
                 if anim_args.color_force_grayscale:
-                    img = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
+                    img = cv2.cvtColor(img.astype(np.uint8), cv2.COLOR_BGR2GRAY)
                     img = cv2.cvtColor(img, cv2.COLOR_GRAY2BGR)
 
                 filename = f"{args.timestring}_{tween_frame_idx:05}.png"

